app/include/zmk/physical_layouts.h

```diff
@@ -49,3 +49,5 @@ int zmk_physical_layouts_revert_selected(void);
 
 int zmk_physical_layouts_get_position_map(uint8_t source, uint8_t dest, size_t map_size,
                                           uint32_t map[map_size]);
+
+int zmk_physical_layouts_get_selected_to_stock_position_map(uint32_t **map);
```

Comment:
It would be good to document that the value written to `*map` is an array, and the return value is its size.

Also, this returns a pointer to a mutable array, but the only changes to this data I'm seeing are done to the static variable, not the return value of this function. Should this return a pointer to const pointer?

Reply:
Added